src/main/resources/avro/variantmethods.avdl

```diff
@@ -8,9 +8,10 @@ import idl "variants.avdl";
 /** This request maps to the body of `POST /variantsets/search` as JSON. */
 record SearchVariantSetsRequest {
   /**
-  The Dataset to search.
+  If specified, will restrict the query to variant sets within the
+  given datasets.
   */
-  string datasetId;
+  array<string> datasetIds = [];
```

Comment:
Because the server expects to parse a one-element array, and not a single string. I'm trying to make the schemas look syntactically as much like the server expects as I can.

If you like, we could specify that the array has to be one element.

If you want to keep it as just a string, the server should parse just a string.